src/main/resources/org/jenkinsci/plugins/saml/SamlEncryptionData/config.jelly

```diff
@@ -18,4 +18,7 @@
     <f:entry title="Auth Request Signature" field="forceSignRedirectBindingAuthnRequest" help="/plugin/saml/help/forceSignRedirectBindingAuthnRequest.html">
         <f:checkbox/>
     </f:entry>
+    <f:entry title="Wants Assertion Signed" field="wantsAssertionsSigned" help="/plugin/saml/help/wantsAssertionsSigned.html">
+        <f:checkbox/>
```

Comment:
```suggestion
        <f:checkbox default="true"/>
```

to retain previous default? (If so, change the `wantsAssertionsSigned` field from `boolean` to `Boolean` and treat `null` like `true` in the getter.)

Reply:
guess not, Jira says

> now is disabled by default

Reply:
yep, the new library causes issues with `wantsAssertionsSigned` enabled.